main.py

```diff
@@ -54,6 +56,15 @@ def get_user(self, id: int) -> BotUser:
         return self.users[id]
 
 
+class Device:
+    def __init__(self, device: dict):
+        self.id: int = int(device["id"])
```

Comment:
Inconsistent quotes. Format code with `black`.

Reply:
I'll consider in a separate change